src/components/core/Main/Main.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
```

Comment:
why do you remove spaces?

Reply:
I think I do not understand why with spaces better. Sorry, I deleted them